frontend/src/constants/API.ts

```diff
@@ -51,6 +51,22 @@ export interface IPropertySearchParams {
 export const PROPERTIES = (params: IPropertySearchParams | null) =>
   `/properties/search?${params ? queryString.stringify(params) : ''}`; // get filtered properties or all if not specified.
 
+export interface IGeoSearchParams {
+  bbox?: string | undefined;
+  address?: string;
+  administrativeArea?: string;
+  pid?: string;
+  projectNumber?: string;
+  agencies?: string; // TODO: Switch to number[]
+  classificationId?: string; // TODO: Switch to number
```

Comment:
these used to be in this format:

classificationId: number | null;
  minLandArea: number | null;
  maxLandArea: number | null;

I can see why you might want to delay fixing agencies, but shouldn't you be able to just keep these all as number?